requirements.txt

```diff
@@ -1,6 +1,7 @@
-pymongo==3.8.0
-requests==2.20.0
 cacheout==0.10.2
-Flask==1.0.2
-gunicorn==19.9.0
-gevent==1.3.5
\ No newline at end of file
+Flask==2.0.0
+gevent==24.2.1
+gunicorn==22.0.0
+pymongo==3.10
+requests==2.20.0
+Werkzeug==2.0.3
```

Comment:
Why do we need a Werkzeug install when we didn't before?

Reply:
when install Flask, Werkzeug is automatically installed. 
I initially upgraded Flask to the latest version 3.0.3, which also installed Werkzeug to 3.0. 
However, `werkzeug.server.shutdown` is deprecated and removed in Werkzeug 2.1
So in 3.0, it never properly shutdown the server for each function, causing the hanging problem.
Therefore I manually downgraded Werkzeug to 2.0.3 fixed the problem.

Reply:
Ok, can you add an issue to find a different way to shut down the server or update the tests so it's not needed so we can update to the most recent version of werkzeug

Reply:
https://github.com/jgi-kbase/IDMappingService/issues/133

Reply:
:+1: